packages/integration-tests/package.json

```diff
@@ -19,6 +19,7 @@
     "type-check": "tsc",
     "pretest": "yarn clean && yarn type-check",
     "test": "playwright test ./suites",
+    "test:replay": "yarn pretest && PWDEBUG=console playwright test ./suites/replay/privacy",
```

Comment:
l: Do we need the pretest here specifically? Otherwise no objections to adding this script :)